app/indices/client_index.rb

```diff
@@ -0,0 +1,3 @@
+ThinkingSphinx::Index.define :client, :with => :real_time do
```

Comment:
always use new notation, and again we will go over your implementation of search tomorrow, i hope you went over the documentation for sphinx.